### DISCUSSION TITLE
[@types/node] Missing "end:boolean"  declaration from PipelineOptions

### DISCUSSION THREAD

**Discussion by Ferrum-Citadel** [DefinitelyTyped/DefinitelyTyped#64038] [Issues with a @types package] [answered]
This was recently implemented in node ( [doc reference ](https://github.com/nodejs/node/blob/762a3a8ad925d56a12b43e0e7f7c811e93097784/doc/api/stream.md?plain=1#L81)) as an available option for the pipeline function of  the 'stream/promises' module.  

The type declaration is missing from [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b2505fdf433904702dc8a5d6c07d06498b1a6ee5/types/node/stream.d.ts#L1082).  

 I haven't contributed before but I could give it a try if its needed.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
note: this was added in v19, was it? The current `@types/node` are still at v18, waiting for update to v19

**Reply by Ferrum-Citadel** [reply]
It was recently merged to main but as far as I can see it was also included in [v.17.20](https://github.com/nodejs/node/pull/40983#issue-1064590493) and in[ v16.14.0](https://github.com/nodejs/node/pull/41804#issue-1120258873).
Kind of a rollercoaster ride tracking that change :sweat_smile:. 
Maybe still out of scope for now.
Thanks for the quick answer!

**Comment by xfournet** [comment, marked as answer]
PR submitted https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65212

**Reply by peterblazejewicz** [reply]
merged
